document/document.mdk

```diff
@@ -45,4 +43,6 @@ Copyright © 2015 Microsoft Corporation. All rights reserved.
 
 [INCLUDE=AnnexC]
 
+[INCLUDE=AnnexD]
+
```

Comment:
Note that I unlinked the Policy Compliance Profile, and no longer link to Annex E.